cogs/tags/tags.py

```diff
@@ -271,33 +283,65 @@ def verify_lookup(self, lookup):
         if len(lookup) > 100:
             raise RuntimeError("Tag name is a maximum of 100 characters.")
 
-    @tag.command(name="max")
+    @tag.group("settings")
     @commands.guild_only()
     @checks.mod_or_permissions()
-    async def max(self, ctx: Context, num_tags: int = None):
-        """Set the max number of tags per user. Leave blank to show current setting.
+    async def settings(self, ctx: Context):
+        """Tag settings."""
+
+    @settings.command(name="max")
+    async def max(self, ctx: Context, role: discord.Role, num_tags: int):
+        """Set the max number of tags per member per role.
+
+        For each member of the specified role, each member will have a maximum
+        number of tags they can create. If the member is part of more than one
+        role, then they will take the MAXIMUM number from the roles that they
+        have.
 
         This limit does not apply to admins or mods.
 
         Parameters:
         -----------
+        role: discord.Role
+            The role to set a maximum for.
         num_tags: int
```

Comment:
Add a way to allow unlimited tags for a role. Maybe using the keyword "infinity" or "unlimited"

Ex: An admin wants to make a "unlimited_tags" role and give it to a handful of users.

Reply:
Hmmm, what if I use the same `NO_LIMIT` constant, and special case that?

Reply:
If NO_LIMIT is infinity then you can just set the limit to NO_LIMIT

Reply:
Sure, done.